wordpress-mobile/WordPress-iOS
# ActivityLog: Discard `isDiscarded`.

Activity Log has done away with the `isDiscarded` property on the backend. Let's do the same on our end. 

This change shouldn't visibly change any app behavior, this is just to make sure we don't forget to get rid of it and then wonder three years down the line "what was that property there ever for!?"

P2 post for context: p9rlnk-ec-p2

To test:
1. Make sure the app builds
1. Open the app
1. Go to Activity Log
1. Make sure nothing explodes
